gempy/library/nddops.py

```diff
@@ -178,24 +155,26 @@ def __init__(self, combine='mean', reject='none', log=None, **kwargs):
             combiner = getattr(self, combine)
             assert getattr(combiner, 'is_combiner')
         except AttributeError:
-            self._logmsg("No such combiner as {}. Using mean instead.".format(combine),
-                         level='warning')
+            self._logmsg("No such combiner as {}. Using mean instead."
+                         .format(combine), level='warning')
             combiner = self.mean
+
         # No combine functions require arguments (yet) but futureproofing
         req_args = getattr(combiner, 'required_args', [])
         self._combiner = combiner
-        self._dict = {k: v for k, v in kwargs.items() if k in req_args}
+        self._comb_args = {k: v for k, v in kwargs.items() if k in req_args}
 
         try:
             rejector = getattr(self, reject)
             assert getattr(rejector, 'is_rejector')
         except AttributeError:
-            self._logmsg('No such rejector as {}. Using none instead.'.format(reject),
-                         level='warning')
+            self._logmsg('No such rejector as {}. Using none instead.'
+                         .format(reject), level='warning')
             rejector = self.none
+
         req_args = getattr(rejector, 'required_args', [])
         self._rejector = rejector
-        self._dict.update({k: v for k, v in kwargs.items() if k in req_args})
+        self._rej_args = {k: v for k, v in kwargs.items() if k in req_args}
```

Comment:
Instead of merging combine and rejection args and then splitting them when they are used, I added two variables.